1-D_Array/Python/Max_And_Next_Max_Number.py

```diff
@@ -0,0 +1,46 @@
+#Python program to find the maximum and the next maximum number in an array
+
+num = int(input("Enter number of elements in array: "))
+arr = []
+print("Enter your array: ");
```

Comment:
Semicolons are not required in Python to terminate a statement as far as I know. You are using them in all files.